src/mdacli/__init__.py

```diff
@@ -7,11 +7,6 @@
 # SPDX-License-Identifier: GPL-2.0-or-later
 """Command line interfaces for MDAnalysis analysis classes."""
 
-from .cli import maincli, setup_clients
+from mdacli.cli import main
```

Comment:
I would remove this from here, and also revert the `setup.py` entry point to `mdacli:cli.main`.